quickstarts/kind.md

```diff
@@ -36,35 +36,34 @@ kubectl version
 If your Docker Daemon runs as a VM you'll most likely need to configure how much memory the VM should have, how many CPUs, how much disk space, and swap size.
 Make sure to assign at least 2 CPUs, and preferably 4 Gb or more of RAM. Consult the Docker documentation for you platform how to configure these settings.
 
-# Starting Kubernetes cluster
+# Starting the Kubernetes cluster
 
-This will start a local development cluster of [Kubernetes Kind](https://github.com/kubernetes-sigs/kind) which installs as a single docker container.
+Start a local development cluster of [Kubernetes Kind](https://github.com/kubernetes-sigs/kind) that installs as a single docker container.
 
 ```shell
 kind create cluster
 ```
 
-# Applying Strimzi installation file
```

Comment:
Everything after this is common between the `kind` and `minikube` docs. Maybe we could use a jekyll `include` to avoid having two files with the same content?

Reply:
I've added a new commit to use the jekyll includes